test/make/windows.mk

```diff
@@ -73,6 +73,16 @@ clean:
 	@echo All generated files removed!
 	@echo.
 
+cleansys:
+	@echo Removing .sys file...
+	@$(DEL) ..\log\windows.sys
+	@echo Removed .sys file!
+	@echo.
+
+cleanall: clean cleansys
+	@echo Removed everything!
+	@echo.
+
 help:
```

Comment:
add cleansys and cleanall descriptions to the help msg

Reply:
Done.